### ISSUE TITLE
Show options without having started typing

### ISSUE THREAD

**Issue by airdev-web** [optimistdigital/nova-multiselect-field#112]
Hello !

Is there a way to display the different options by default without having to write the first letters of the search first?

I use `->belongsToMany(Resource::class)` and my list contains only 10 elements. So it's faster to see the list directly than to have to search.

Thank you!

**Comment by Tarpsvo** [comment]
Yeah, the second parameter of `->belongsToMany` allows you to toggle between async and sync.

You can just use
```
->belongsToMany(Resource::class, false)
```

**Comment by ekandreas** [comment]
When using the api this is not the case...

**Comment by DuyenNguyen66** [comment]
> When using the api this is not the case...

I got same issue when using the api in field. Any update?
